.gitlab/download-serverless-artifacts.sh

```diff
@@ -32,41 +47,31 @@ fi
 
 echo "Found build with id '$buildId' for commit '$CI_COMMIT_SHA' on branch '$branchName'"
 
-architectures=("x64" "arm64")
```

Comment:
As we do the aggregation in azdo, we don't need to build the list of artifacts here, we just pull a single artifact so, this code becomes almost identical the SSI code (which is handy)

Reply:
IT looks like a big change, but it's _mostly_ just deleing the for loops - hiding whitespace changes makes the changes more obvious